Replace the ruby program prefix with the target ditto, so that ruby
modules are installed inside the target

Signed-off-by: Jonas Karlsson <jonas@gobolinux.org>
--- qt4-qtruby-1.4.9/cmake/modules/FindRUBY.cmake	2008-03-24 21:25:36.000000000 +0100
+++ qt4-qtruby-1.4.9/cmake/modules/FindRUBY.cmake.new	2008-03-24 22:08:15.000000000 +0100
@@ -53,6 +53,8 @@
 	STRING(REPLACE "\n" "" RUBY_VERSION "${RUBY_VERSION}")
 	STRING(REPLACE "\n" "" RUBY_SITE_LIB_DIR "${RUBY_SITE_LIB_DIR}")
 	STRING(REPLACE "\n" "" RUBY_SITE_ARCH_DIR "${RUBY_SITE_ARCH_DIR}")
+	STRING(REPLACE "@%Compile_ruby_path%@" "@%Compile_target%@" RUBY_SITE_LIB_DIR "${RUBY_SITE_LIB_DIR}")
+	STRING(REPLACE "@%Compile_ruby_path%@" "@%Compile_target%@" RUBY_SITE_ARCH_DIR "${RUBY_SITE_ARCH_DIR}")
 
 	FIND_PATH(RUBY_INCLUDE_PATH 
   		NAMES ruby.h
